manim/mobject/mobject.py

```diff
@@ -225,7 +225,17 @@ def construct(self):
 
     @property
     def animate(self):
-        """Used to animate the application of a method.
+        """Used to animate the application of any method of :code:`self`.
+
+        Any method called on :code:`animate` is converted to an animation that repeatedly calls that method on the mobject.
+
+        For example, :code:`square.set_fill(WHITE)` sets the fill color of a square, while :code:`sqaure.animate.set_fill(WHITE)` animates this action.
+
+        Multiple methods can be put in a single animation once via chaining:
```

Comment:
This feature was previously only mentioned in a warning. Warnings should be supplementary to documentation, not hiding extra features.

Reply:
👍